libs/features/pages/home/src/lib/features-pages-home.module.ts

```diff
@@ -4,9 +4,10 @@ import { CommonModule } from '@angular/common';
 import { HomePageComponent } from './main/home/home-page.component';
 
 import { HomeRoutingModule } from './home.routing';
+import { ElementsCardsModule } from '@elewa-website/elements/cards';
 
```

Comment:
This changes should not be pushed. Any component that needs this will import it.